.azure-pipelines/testcases_collection/testcases.collection.yml

```diff
@@ -0,0 +1,39 @@
+name: TestCasesCollection_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+pr: none
+
+schedules:
+  # Run at GME+4 everyday
+  - cron: "0 4 * * *"
+    displayName: TestCases Collection
+    branches:
+      include:
+        - master
+    always: true
+
+stages:
+
+- stage: TestCasesCollection
+  jobs:
+    - job: TestCasesCollection
+      variables:
+        - group: KUSTO_SECRETS
+
+      steps:
+        - task: Bash@3
+          displayName: TestCases Collection
+          inputs:
+            targetType: 'inline'
+            script: |
+              set -x
+
+              cd tests
+              pip install -r requirements.txt
```

Comment:
This is to use the `tests/requirements.txt`?

Reply:
No, we have requirments.txt under folder `testcases_collection`. We will use this.